packages/ruby/README.md

```diff
@@ -66,6 +66,19 @@ config.middleware.use Readme::Metrics, options do |env|
     email: current_user.email
   }
 end
+
+```
+
+#### Content-Length
+
+In Rails 5 the `Rack::ContentLength` middleware was [removed from the default
+middleware
+stack](https://github.com/rails/rails/commit/56903585a099ab67a7acfaaef0a02db8fe80c450).
+If you'd like to include the content length of your applications responses in
+your requests to ReadMe, you'll need to add it to your middleware:
+
+```ruby
+config.middleware.use Rack::ContentLength
```

Comment:
I like this documentation! 👍 

However, I imagine that users of the middleware don't care about this. On the other hand, it feels useful to let them know about this distinction? I'm pulled in two directions here.

Reply:
This seems useful to me to include. You and I were both surprised that the content length header was missing so I'm sure other folks might be as well.

Reply:
I removed this instructions now that we'll always have a content-length value from either the header or one we've calculated.